### ISSUE TITLE
npm run dev fails out of the box

### ISSUE THREAD

**Issue by daveroberts** [keithamus/npm-scripts-example#33]
```
$ git clone https://github.com/keithamus/npm-scripts-example.git
$ npm install
$ npm run dev
```

Produces the following output

```
> npm-scripts-example@1.0.0 dev /apps/home/daver/code/npm-scripts-example
> npm run open:dev -s && parallelshell "npm run live-reload -s" "npm run serve -s" "npm run watch -s"

Starting up http-server, serving dist/ on: http://0.0.0.0:9090
Hit CTRL-C to stop the server
live reload server listening on port 9091 reloading on files dist/
/apps/home/daver/code/npm-scripts-example/node_modules/nodemon/lib/config/exec.js:158
  extension = extension.match(/\w+/g).join(',');
                                     ^

TypeError: Cannot read property 'join' of null
    at exec (/apps/home/daver/code/npm-scripts-example/node_modules/nodemon/lib/config/exec.js:158:38)
    at /apps/home/daver/code/npm-scripts-example/node_modules/nodemon/lib/config/load.js:56:29
    at callback (/apps/home/daver/code/npm-scripts-example/node_modules/nodemon/lib/config/load.js:161:5)
    at ReadFileContext.callback (/apps/home/daver/code/npm-scripts-example/node_modules/nodemon/lib/config/load.js:171:14)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:303:13)
```

**Comment by daveroberts** [comment]
It looks like this happening with nodemon, I filed a separate report there, but the issue should be addressed here as well.

https://github.com/remy/nodemon/issues/743

**Comment by francoismassart** [comment]
Is it fixed since https://github.com/remy/nodemon/issues/743 was closed with the release of `nodemon@1.14.0` ?

**Comment by francoismassart** [comment]
I updated `nodemon` yesterday if you want to test it out. 

I did not get errors running `dev` on my side...
